flake8_quotes/__init__.py

```diff
@@ -12,31 +12,53 @@ class QuoteChecker(object):
     QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
+        # When user wants only single multiline quotes
+        '\'': {
+            'good': '\'\'\'',
+            'bad': '"""'
+        },
+
+        # When user wants only double multiline quotes
+        '"': {
+            'good': '"""',
+            'bad': '\'\'\''
+        }
+    }
+
+    MULTILINE_QUOTES_STRINGS = ('\'\'\'', '"""')
+
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append('quotes')
+        parser.add_option('--multiline-quotes', action='store',
+                          help='Multiline quote to expect in all files (disabled by default)')
+        parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        if hasattr(options, 'multiline_quotes'):
```

Comment:
Is there any reason this is `False` apart from the tests? In that case the tests should provide it (see my patch #33), as the production code shouldn't be modified just because of tests and instead the tests should be modified (if possible and reasonable).